[layout] widget_layout.h is missing

I got an error message when build.

```
/usr/include/LCUI/gui/widget.h:47:10: fatal error: LCUI/gui/widget_layout.h: No such file or directory
 #include <LCUI/gui/widget_layout.h>
```

**Reproduction Steps:**

1. `./autogen.sh`
2. `./configure --prefix=/usr`
3. `make && make install`
4. Build the code below.

```c
#include <LCUI_Build.h>
#include <LCUI/LCUI.h>
#include <LCUI/gui/widget.h>
#include <LCUI/gui/widget/textview.h>

int
main (int argc, char *argv[])
{
  (void)argc;
  (void)argv;

  LCUI_Widget root, text;

  LCUI_Init ();
  root = LCUIWidget_GetRoot ();
  text = LCUIWidget_New ("textview");
  TextView_SetText (text, "Hello World!");
  Widget_Append (root, text);

  return LCUI_Main ();
}
```

**LCUI version:** v1.0-beta
**OS and version:** Linux
**Build tools:** gcc 7.3.0

--- a/include/LCUI/gui/Makefile.am
+++ b/include/LCUI/gui/Makefile.am
@@ -4,5 +4,5 @@ SUBDIRS=widget
 pkginclude_HEADERS = widget_base.h widget_task.h widget_prototype.h \
 widget_style.h widget_event.h widget_paint.h widget.h css_library.h \
 widget_helper.h css_parser.h css_rule_font_face.h css_fontstyle.h \
-builder.h metrics.h
+builder.h metrics.h widget_layout.h
 pkgincludedir=$(prefix)/include/LCUI/gui

